images/metering-ansible-operator/roles/meteringconfig/tasks/reconcile.yml

```
@@ -12,6 +12,78 @@
     hiveStorageType: "{{ meteringconfig_spec_overrides | json_query('storage.hive.type') }}"
   when: hiveStorageType is undefined or hiveStorageType not in ['s3', 'sharedPVC', 'hdfs']
 
+- name: Check if we need to create a Presto TLS secret using the user-populated fields
+  set_fact:
+    meteringconfig_reporting_operator_server_ca_certificate: "{{ _presto_spec.presto.config.tls.caCertificate }}"
```

So, none of our roles should need to use the `_` variables, they should generally just all consuming the `meteringconfig_spec` variable, and we should be setting the variables within `meteringconfig_spec` such that it's all we need.

These `_` variables are only really to make things less verbose within the default variables file, not for use outside that file. So in this case, I'd expect you to have:

```suggestion
    meteringconfig_reporting_operator_server_ca_certificate: "{{ meteringconfig_spec.presto.spec.presto.config.tls.caCertificate }}"
```